deploy/test-environments/fleet_api/src/configuration_fleet.py

```diff
@@ -31,3 +31,10 @@
 aws_config = Munch()
 aws_config.access_key_id = os.getenv("AWS_ACCESS_KEY_ID", "NA")
 aws_config.secret_access_key = os.getenv("AWS_SECRET_ACCESS_KEY", "NA")
+
+gcp_dm_config = Munch()
+gcp_dm_config.deployment_name = os.getenv("DEPLOYMENT_NAME", "")
+gcp_dm_config.zone = os.getenv("ZONE", "us-central1-a")
+gcp_dm_config.allow_ssh = bool(
+    os.getenv("ALLOW_SSH", "false") == "true",
+)
```

Comment:
Already a boolean, no?
```suggestion
gcp_dm_config.allow_ssh = os.getenv("ALLOW_SSH", "false") == "true"
```